async.md

```diff
@@ -228,6 +228,46 @@ Collection.createUniqueIndex(columnName)
  * @returns {Promise<>}
  */
 Collection.deleteCollection()
+
+/**
+ * @typedef {Object} ContinuousAggregateOptions
+ * @prop {string} view_name: The name of the continuous aggregate
+ * @prop {string} start_offset: The offset from the current date to begin aggregating data
```

Comment:
what format should these strings be? are they durations like "12h" or something?